chaosk8s/node/actions.py

```diff
@@ -22,7 +22,8 @@
 def _select_nodes(name: str = None, label_selector: str = None,
                   count: int = None, secrets: Secrets = None,
                   pod_label_selector: str = None,
-                  pod_namespace: str = None, first=False):
+                  pod_namespace: str = None, first: bool =
+                  False) -> list[client.V1Node:]:
```

Comment:
This should likely be `List[client.V1Node]:` rather than `list[client.V1Node:]:` which is not a valid syntax. The `List` comes from the |typing](https://docs.python.org/3/library/typing.html#module-typing) module.